### ISSUE TITLE
/ngx_pagespeed_statistics not loading

### ISSUE THREAD

**Issue by Vilial** [pagespeed/ngx_pagespeed#1371]
The exact same problem is described [here](http://serverfault.com/questions/810454/ngx-pagespeed-admin-statistics-dont-load)

"Sorry, failed to update the statistics. Please wait and try again later."
a request to /ngx_pagespeed_statistics/stats_json returns 404

The solution mentioned [here](http://community.rtcamp.com/t/ngx-pagespeed-not-working-properly/5911/8)  doesn't work. Maybe since my website doesn't have such thing as root folder...

**Comment by oschaaf** [comment]
After looking in to this, I am wondering if you are able to access the statistics by navigating to it via `/pagespeed_admin/statistics` .
If you access the statistics through that url, it should request `stats_json` from `/pagespeed_admin/stats_json` instead (assuming you have set things up according to the docs).
Does that work?

**Comment by Vilial** [comment]
Yes, /pagespeed_admin/statistics works and successfully requests /pagespeed_admin/stats_json

**Comment by oschaaf** [comment]
Thanks for confirming
